src/test/unit/project-converter_test.ts

```diff
@@ -74,7 +74,7 @@ suite('AnalysisConverter', () => {
       packageType: PackageType;
       npmImportStyle: NpmImportStyle;
       expectedWarnings: string[];
-      includes: string[];
+      entrypoints: Map<string, OriginalDocumentUrl[]>;
```

Comment:
I have one last idea to try to get rid of all the `OriginalDocumentUrl` casting: can we not extend `PartialConversionSettings` here, and then set `entrypoints` to `Map<string, string[]>`? We should still get proper type warning if we set bad properties on `partialSettings`.